tests/e2e/test_rosacli_version.go

```diff
@@ -0,0 +1,54 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/rosa/pkg/info"
+	"github.com/openshift/rosa/tests/ci/labels"
+	"github.com/openshift/rosa/tests/utils/exec/rosacli"
+)
+
+var _ = Describe("Get CLI version",
+	labels.Day1,
+	labels.FeatureVersion,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			rosaClient *rosacli.Client
+		)
+
+		BeforeEach(func() {
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+		})
+
+		It("can get the version of rosa CLI while logged out - [id:73743]",
```

Comment:
This test doesn't work on older versions of ROSA CLI, right? Should we mark anywhere to skip this test on pre-1.2.39 versions or whatever the number is? I don't know if we have test clusters running on older versions of the software so maybe this is overkill

Reply:
This did work on older versions, I think we regressed in the .38 timeframe  or so.

Reply:
I'm a bit unclear as to what the behavior is pre-1.2.39. My understanding from the comments on the jira ticket is that it only affects builds from the master branch pre-1.2.39, but 1.2.38 works.

Also, this test assumes that the installed version of `rosa` is the one that matches the codebase as it pulls the version number from the code

Reply:
sounds like from what mike's saying that the code as is should be good to go, so LGTM!